CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
 
 #define project name
-PROJECT(ryzenadj)
+PROJECT(ryzenadj VERSION 0.11.1)
```

Comment:
We have to be careful here to not miss this during version increment. Is there a way to not forget this part during version increment?

Reply:
Can see if it would be possible via git and searching for the closest tag. Dont know if I can execute a custom command before the project tag. Need to have a look.  Same thing is true for the `debian/changelog` file. I.E need to add an entery modify before you build. Above versioning is for the shared library simlinks.

Reply:
Or probably we can use cmake as main source of version number. Define preprocessor macro or just do string insertion into the header.

Reply:
Added automatic header generation. Also changed the github CI/CD build to auto change the Debian version on build using git describe. I.E don't need to manually update the `debian/changelog` file.